packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx

```diff
@@ -139,8 +139,8 @@ describe("MafsGraph", () => {
         };
 
         render(<MafsGraph {...basePropsWithTexLabels} />);
-        expect(screen.getByText("1/2")).toBeInTheDocument();
-        expect(screen.getByText("3/4")).toBeInTheDocument();
+        expect(screen.getByText("\\text{$1/2$}")).toBeInTheDocument();
+        expect(screen.getByText("\\text{$3/4$}")).toBeInTheDocument();
```

Comment:
@anakaren-rojas I think my update affected some of your recent changes. Does this look okay to you?

Reply:
For these, I think it would be 
```suggestion
expect(screen.getByText("$\\frac{3}{4}$")).toBeInTheDocument();
```

Reply:
`$\frac{1}{2}$` renders correctly when I test it out in storybook, so the functionality updates you made look right